test/repositories/ContractEntityRepository/util.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 OpenST Ltd.
```

Comment:
Let's remove this till it's clarified if license is needed or not.
Either we add to all files or add to none.

Reply:
Removed from files in the PR.